tests/test.py

```diff
@@ -23,6 +26,15 @@ def setUpClass(cls):
         cls.logger = _utils.get_logger('dbt-invoke', level='DEBUG')
         cls.config_path = Path(PARENT_DIR, 'test_config.yml')
         cls.config = _utils.parse_yaml(cls.config_path)
+
+        # for backward compatibility, select the correct dbt_project.yml file
+        if pkg_resources.get_distribution("dbt-core").version >= '1.0.0':
+            shutil.copy(Path(PARENT_DIR, 'dbt_project_files/dbt_project.yml'),
+                        Path(PARENT_DIR, cls.config['project_name'], 'dbt_project.yml'))
+        else:
+            shutil.copy(Path(PARENT_DIR, 'dbt_project_files/dbt_project_pre_dbt_v1.yml'),
+                        Path(PARENT_DIR, cls.config['project_name'], 'dbt_project.yml'))
+
         cls.project_dir = Path(PARENT_DIR, cls.config['project_name'])
```

Comment:
Moving the definition of `cls.project_dir` above this section, so we don't have to say `Path(PARENT_DIR, cls.config['project_name']...` more than once.

```suggestion
        cls.project_dir = Path(PARENT_DIR, cls.config['project_name'])
        
        # for backward compatibility, select the correct dbt_project.yml file
        if pkg_resources.get_distribution("dbt-core").version >= '1.0.0':
            shutil.copy(Path(PARENT_DIR, 'dbt_project_files/dbt_project.yml'),
                        Path(cls.project_dir, 'dbt_project.yml'))
        else:
            shutil.copy(Path(PARENT_DIR, 'dbt_project_files/dbt_project_pre_dbt_v1.yml'),
                        Path(cls.project_dir, 'dbt_project.yml'))

```